src/lib/get-topper-settings.js

```diff
@@ -213,7 +218,7 @@ const getTopperSettings = (content, flags = {}) => {
 	} else if (isPackage(content)) {
 		return usePackageTopper(content);
 	} else if (isEditoriallySelected(content)) {
-		return useEditoriallySelectedTopper(content);
+		return useEditoriallySelectedTopper(content, flags);
```

Comment:
I don't think you need this change anymore

```suggestion
		return useEditoriallySelectedTopper(content);
```